src/Controls/src/Core/Platform/Android/Extensions/ToolbarExtensions.cs

```diff
@@ -100,7 +105,13 @@ public static void UpdateBackButton(this AToolbar nativeToolbar, Toolbar toolbar
 			else
 			{
 				if (!toolbar.DrawerToggleVisible)
+				{
 					nativeToolbar.NavigationIcon = null;
+				}
+				else
+				{
+					nativeToolbar.SetNavigationContentDescription(Resource.String.nav_app_bar_open_drawer_description);
```

Comment:
Can you create an issue for this?  The `Toolbar` hasn't caught up yet to all the features of shell. We'll need to add a property to `Toolbar.cs` so that this is settable from the extension methods. `ShellToolbarTracker` will set these values based on user settings when using `Shell` but we'll need to expose something for non-shell apps

Reply:
https://github.com/dotnet/maui/issues/5762